src/nvim/mbyte.h

```diff
@@ -28,6 +28,18 @@
 #define ENC_LATIN9     0x400       /* Latin9 */
 #define ENC_MACROMAN   0x800       /* Mac Roman (not Macro Man! :-) */
 
+// TODO(bfredl): eventually we should keep only one of the namings
```

Comment:
Agree, it is also matches the name of the module/feature ("multibyte"), and utf is an awkward abbreviation for utf8 :)